test/units/oracle-transport-ssb.sh

```diff
@@ -1,55 +1,59 @@
-#!/usr/bin/env bash
-TEST_PATH=$(cd "${BASH_SOURCE[0]%/*}" && pwd)
+#!/bin/bash
+test_path=$(cd "${BASH_SOURCE[0]%/*}"; pwd)
```

Comment:
this is probably fine. I tend to use the POSIX `$0` and got curious about the differences here... which led me to https://stackoverflow.com/a/68359914/183677 which says you can have both :P

```
[${BASH_SOURCE[0]:-$0}]
```